Dockerfile

```diff
@@ -5,4 +5,4 @@ RUN dnf update -y && dnf install -y python3 python3-pip && dnf clean all && pyth
 WORKDIR assisted_event_scrap/
 ADD .  ./
 COPY --from=service /clients/assisted-service-client-*.tar.gz /build/pip/
-RUN python3 -m pip install -r requirements.txt && python3 -m pip install --upgrade /build/pip/* && rm -rf build/
+RUN python3 -m pip install -r requirements.txt vcversioner && python3 -m pip install --upgrade /build/pip/* && rm -rf build/
```

Comment:
why separating ``vcversioner`` from ``requirements.txt``?

Reply:
It required for build only, not running the project